docker/Dockerfile

```diff
@@ -27,6 +27,8 @@ RUN \
     mkdir -p /usr/lib/presto /data/presto && \
     chown -R "presto:presto" /usr/lib/presto /data/presto
 
+ENV LANG en_US.UTF-8
```

Comment:
I'd move this to bottom, before CMD

also, did you consider LC_ALL, LANGUAGE (https://stackoverflow.com/a/28406007/65458)?

(i think you can set multiple env vars in single `ENV`  build command, so still one additional layer, not many)

Reply:
According to [POSIX](https://pubs.opengroup.org/onlinepubs/7908799/xbd/envvar.html):

> LANG ... This variable determines the locale category for native language, local customs and coded character set in the absence of the LC_ALL and other LC_*

Per [locale(7)](http://man7.org/linux/man-pages/man7/locale.7.html) the `LANGUAGE` variable seems to be specific to gettext :

> The GNU gettext family of functions also obey the environment variable LANGUAGE

So setting `LANG` alone seems appropriate.